rust/api/src/lib.rs

```diff
@@ -1 +1,3 @@
+pub mod error;
```

Comment:
Could you add `#![deny(rustdoc::broken_intra_doc_links)]` here and in other crates. Otherwise, there's a lot of broken doc links that are not being caught.